llama_stack/cli/model/model.py

```diff
@@ -35,3 +36,4 @@ def __init__(self, subparsers: argparse._SubParsersAction):
         ModelPromptFormat.create(subparsers)
         ModelDescribe.create(subparsers)
         ModelVerifyDownload.create(subparsers)
+        ModelRemove.create(subparsers)
```

Comment:
Could you help update the CLI reference doc to include the added support? https://github.com/meta-llama/llama-stack/blob/main/docs/source/references/llama_cli_reference/index.md